src/Microsoft.ML.TensorFlow/TensorFlow/Tensorflow.cs

```diff
@@ -2238,4 +2244,19 @@ public TFTensor AsTensor()
             return shape.AsTensor();
         }
     }
+
+    internal class TFString
```

Comment:
>TFString [](start = 19, length = 8)

Consider importing these APIs directly in the TFTensor class, this way they can be private.